statics/js/components/preference.js

```diff
@@ -0,0 +1,98 @@
+/* jshint multistr: true */
+
+var PreferenceComponent = {
+
+  name: 'Preference',
+
+  mixins: [notificationMixin],
+
+  template: '\
+    <form class="form-preference" @submit.prevent="save">\
+      <h2> Preferences </h2>\
+      <div class="form-group">\
+        <label for="collapse-index">Collapse Index</label>\
```

Comment:
It could be nice to add another line below this one with explanation of the parameter or to add a "?" picto on the right of the label displaying a tooltip.

Reply:
added a help text